src/Symfony/Component/Form/Extension/Core/Type/CollectionType.php

```diff
@@ -116,17 +88,11 @@ public function configureOptions(OptionsResolver $resolver)
             'prototype' => true,
             'prototype_data' => null,
             'prototype_name' => '__name__',
-            // deprecated as of Symfony 2.8, to be removed in Symfony 3.0. Use entry_type instead
-            'type' => null,
-            // deprecated as of Symfony 2.8, to be removed in Symfony 3.0. Use entry_options instead
-            'options' => null,
-            'entry_type' => $entryType,
-            'entry_options' => $entryOptions,
+            'entry_type' => __NAMESPACE__.'\TextType',
```

Comment:
If we do that, we probably want to switch to the class constant everywhere in the form component.